BUILD.bazel

```diff
@@ -73,13 +87,19 @@ cc_library(
     includes = ["src/"],
     linkopts = LINK_OPTS,
     visibility = ["//visibility:public"],
+    # In Bazel 6.0+, these will be `interface_deps`:
     deps = [
         ":arena",
         "//src/google/protobuf/io",
         "//src/google/protobuf/stubs:lite",
     ],
 )
 
+# `:internal_protobuf_full` is an internal target which has the non-lite
```

Comment:
Should this go above "internal_protobuf_full"?

Reply:
Same... removed.